### PR TITLE
[tests-only] Update webUI test suites in drone

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Updating webUI test suite names

## Related Issue
- Closes https://github.com/owncloud/ocis/issues/2029

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by Talank — COMMENTED** [review]
LGTM

**Review by swoichha — COMMENTED** [review]
lgtm

File: .drone.star [hunk 1]
```diff
@@ -46,42 +46,47 @@ config = {
             ],
             "webUISharingBasic": [
                 "webUISharingAcceptShares",
-                "webUISharingAcceptSharesToRoot",
```
Comment (phil-davis) [inline .drone.star:49]: I guess you are deleting this because "sharing to root"  is not a valid thing to do in OCIS.

File: .drone.star [hunk 2]
```diff
@@ -46,42 +46,47 @@ config = {
...
             ],
             "webUIRestrictSharing": "webUIRestrictSharing",
             "webUISharingNotifications": [
                 "webUISharingNotifications",
-                "webUISharingNotificationsToRoot",
```
Comment (phil-davis) [inline .drone.star:54]: I guess you are deleting this because "sharing to root"  is not a valid thing to do in OCIS.

**Review by phil-davis — APPROVED** [review]
LGTM

**Pull request merged**